Assets/_BForBoss/_Character/Scripts/PlayerMovementBehaviour.cs

```diff
@@ -68,6 +68,17 @@ public void SetControlConfiguration()
             characterLook.controllerHorizontalSensitivity = _inputConfiguration.ControllerHorizontalSensitivity;
             characterLook.controllerVerticalSensitivity = _inputConfiguration.ControllerVerticalSensitivity;
         }
+
+        public void ModifyPlayerSpeed(int modificationPercentage, out float originalPlayerSpeed)
+        {
+            originalPlayerSpeed = maxWalkSpeed;
```

Comment:
You should keep an `_originalPlayerSpeed` within this script instead of outside. 

Having it outside means having 2 sources of truth of "original speed" and get confusing.